src/ui/common/attributes/AttributeRole.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Row, Col, FormGroup } from 'patternfly-react';
+import { FieldArray } from 'redux-form';
+import RoleSelectRenderer from 'ui/common/form/RoleSelectRenderer';
+
+class AttributeRole extends Component {
+  componentWillMount() {
+    const { onWillMount, ...allprops } = this.props;
+    onWillMount(allprops);
+  }
```

Comment:
Maybe change to componentDidMount() ?

https://daveceddia.com/where-fetch-data-componentwillmount-vs-componentdidmount/